portal-impl/src/META-INF/util-spring.xml

```diff
@@ -117,7 +117,6 @@
 	<bean class="com.liferay.portal.kernel.portlet.configuration.icon.PortletConfigurationIconMenu" id="com.liferay.portal.kernel.portlet.configuration.icon.PortletConfigurationIconMenu" />
 	<bean class="com.liferay.portal.kernel.portlet.configuration.icon.PortletConfigurationIconTracker" id="com.liferay.portal.kernel.portlet.configuration.icon.PortletConfigurationIconTracker" />
 	<bean class="com.liferay.portal.kernel.portlet.toolbar.PortletToolbar" id="com.liferay.portal.kernel.portlet.toolbar.PortletToolbar" />
-	<bean class="com.liferay.portal.kernel.sanitizer.SanitizerUtil" id="com.liferay.portal.kernel.sanitizer.SanitizerUtil" />
```

Comment:
@tinatian Hi Tina,
I'm not sure why this SanitizerUtil is still in here since Spring is not instantiating the impls anymore.
both SanitizerImpls are components in modules.
If this is removed from util-spring, do I still need to add this Util class to SF OSGiReferencecheck (on my last commit [here](https://github.com/julschong/liferay-portal/pull/492/commits/12870d1125126d28a2f0e54993728c61feebda30))?